### ISSUE TITLE
LGTM.com - false positive with torch.nn.Sequential

### ISSUE THREAD

**Issue by ekolve** [github/codeql#5789]
An alert was triggered that a [torch.nn.Sequential](https://pytorch.org/docs/stable/generated/torch.nn.Sequential.html) model was not callable.

The model is intended to be a callable - https://pytorch.org/tutorials/beginner/pytorch_with_examples.html#pytorch-nn.

Alert:
https://lgtm.com/projects/g/allenai/allenact/snapshot/5a2b317d0532bd2999b6bfa101522ff45c7ea8c9/files/allenact/embodiedai/mapping/mapping_models/active_neural_slam.py?sort=name&dir=ASC&mode=list#x3719db148fcfd007:1

**Comment by RasmusWL** [comment]
I agree that this looks like a false positive, since the base class `torch.nn.modules.Module` has `__call__` [defined](https://github.com/pytorch/pytorch/blob/0df574017d8f0eb1527e973defb9c65a5befe966/torch/nn/modules/module.py#L1101). Thank you for reporting it!

Our current focus is on improving our security analysis. Because your report does not relate to a security query, we will put this on our backlog and prioritize it if we get enough reports of the same underlying issue in other projects. If you think that your report is related to our security analysis, please clarify that in a comment. Either way, we'll let you know here as soon as it's fixed!

Just FYI, all of our queries are [open source](https://github.com/github/codeql), so if you do have an idea for a code change, we encourage you to open a pull request. I also want to point out that both GitHub Code Scanning and LGTM.com have facilities for suppressing individual alerts or disabling a query.
